cmake/templates/conf.py.in

```diff
@@ -40,6 +40,7 @@ sys.path.insert(0, os.path.abspath('.') + '/extensions')
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    'sphinxcontrib.bibtex',
```

Comment:
We most likely will have to install this module for our CI environment here: https://github.com/STEllAR-GROUP/docker_build_env/blob/master/hpx_build_env/Dockerfile#L94. Would you mind creating another PR over there as well?

Reply:
Thought so, [done](https://github.com/STEllAR-GROUP/docker_build_env/pull/43)!